metadata.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "puppet-mysql_java_connector",
-  "version": "1.0.0",
-  "author": "Merritt Krakowitzer",
```

Comment:
perhaps a comment in the changelog? contributors? readme??
that this module was originally written by Merritt